### ISSUE TITLE
en_US.UTF-8 locale is not installed on all systems by default

### ISSUE THREAD

**Issue by SmaugPool** [cardano-community/guild-operators#307]
`C.UTF-8` most likely is, but we would lose the thousand separators.

**Comment by Scitz0** [comment]
Adding en_US.UTF-8 as a requirement is most likely the best solution. Doesn't have to be set, just installed.

**Comment by SmaugPool** [comment]
Just to list alternatives, using
```
$ numfmt --format "%'.6f" --to-unit=M 1000000000000
1 000 000,000000
```
for all ADA values works also fine now without setting a locale.

Benefits:
* lovalaces always have 6 decimals, which is not bad for their legibility in my opinion
* no LOCALE setting needed, and the user locale is respected

Drawbacks:
* Some locales use an unicode character for the thousands separator. This works fine for now, but this could lead to new issues in the future potentially

**Comment by Scitz0** [comment]
If we decide to always output 6 decimals this basically removes the need to output/store values in both Lovelace and ADA.
The reason ADA was added was to make it more readable but might not be needed. Haven't made up my own mind on this yet.

**Comment by Scitz0** [comment]
Fixed in v2.0.0 that's to be released soon.
Replaced with `numfmt --format "%'.6f" --to-unit=M `
The amount now showed in this format everywhere instead of Lovelace in some places and ADA in some.
